packages/types/lib/testLine.d.ts

```diff
@@ -142,6 +142,10 @@ export type BrowserCommandTestLine = BaseTestLine & {
 export type ElementTarget = {
 	type: 'element',
 	elementId: string,
+	coordinates?: {
```

Comment:
Pls, add translation for such lines and test coverage

Reply:
Done.